web/src/components/studies/Studies.tsx

```diff
@@ -51,6 +51,7 @@ export default function Studies(props: Props) {
         additional_study_admin_usernames: data.additionalStudyAdminUsernames
           .map((admin) => admin.value.trim())
           .map((username) => `${username}${domainName}`),
+        has_existing_data: data.hasExistingData === "true", // Convert to boolean
```

Comment:
Since the radio button stores data asa string, it is converted here to make it a boolean. But we can think more about the data type if needed.

Reply:
I considered both. In the end I though having two very explicit options to choose from would be more clear to the user. But happy to rethink this if there are other ideas?